app/routes/ui/scenario.py

```diff
@@ -63,6 +63,7 @@ async def get(self, scenario_friendly_name, scene_number=0):
                 + "-navmesh"
                 + ".gltf"
             )
+            cursor_mode = int(self.get_argument("cursor_mode", "1"))
```

Comment:
We should catch the `ValueError` exception here and throw a generic exception message here, because otherwise whatever the user writes will be reflected back in HTML, which isn't good security practice usually.
Example: If you go to http://localhost:8888/teacher-escape-room/0?cursor_mode=abc, you will see `abc` reflected in the HTML page